packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.test.tsx

```diff
@@ -104,51 +147,65 @@ function createTestProps(
 }
 
 describe('CombinedRangeFacetFilter tests', () => {
+  beforeAll(() => server.listen())
+  beforeEach(() => {
+    jest.clearAllMocks()
+    server.use(...getHandlersForTableQuery(mockQueryResponseData))
+  })
+  afterEach(() => server.restoreHandlers())
+  afterAll(() => server.close())
+
   let props: CombinedRangeFacetFilterProps
-  function init(overrides?: CombinedRangeFacetFilterProps) {
+  async function init(overrides?: CombinedRangeFacetFilterProps) {
```

Comment:
Now that we run a query, change initialization to wait for loading state to finish before looking for the expected state